exchanges/margin/margin.go

```diff
@@ -23,6 +23,11 @@ func (t *Type) UnmarshalJSON(d []byte) error {
 	return err
 }
 
+// MarshalJSON conforms type to the Marshaler interface
```

Comment:
Nit:
```suggestion
// MarshalJSON conforms type to the json.Marshaler interface
```
I'd also argue for `implements` over `conforms`, but that's getting picky.